packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -272,7 +276,7 @@ class Auth {
             .then(async () => {
                 const token = await fn()
                 this.handleTokenResponse(token, isGuest)
-
+                onClient() && (await this.createOCAPISession())
```

Comment:
I think it makes more sense to put the condition here

```suggestion
                if (onClient() && this.OCAPISessionsURL) {
                    await this.createOCAPISession())
                }
```

Reply:
I wonder, do we need the `await`? We don't need the result to do anything right?